lib/cdo/contact_rollups.rb

```diff
@@ -93,15 +93,15 @@ def self.mysql_multi_connection
     course2
     course3
     course4
-    coursea-2017
-    courseb-2017
-    coursec-2017
-    coursed-2017
-    coursee-2017
-    coursef-2017
+    coursea-2018
+    courseb-2018
+    coursec-2018
+    coursed-2018
+    coursee-2018
+    coursef-2018
```

Comment:
Do we actually want to replace the `-2017` versions, or just add the `-2018` ones? After all, it looks like we had courses 1-4 and A-F living side-by-side here.

Reply:
Yup.  Currently Contact Rollups collects all information about contacts past and present each time it runs.  I think we need to gather information about teachers associated with all script versions.  Also ... is there a way to gather this list from Script Constants instead of duplicating those lists here?

Reply:
good question @sureshc . If you just want to add 2018 versions, then you can pull in csf_2018 from script_constants.rb. 

When will you want to start pulling in 2019 courses also? It looks like `csf_2019` has not yet been created in script_constants.rb yet, but now could be a good time to create that list so that you don't have to maintain it here. 

Whatever you decide, a comment on this array saying what should or shouldn't be in here would be nice so we don't have to reverse engineer this all again next year. For example, if you want to include 2019 courses once they become visible, stable, etc then that should be noted here.